src/app/identity-status/middleware.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from "express";
+import { NextFunction, RequestHandler } from "express-serve-static-core";
+import { getDcMawPoll } from "../../services/coreBackService";
+
+enum IdentityProcessingStatus {
+  COMPLETED = "COMPLETED",
+  ERROR = "ERROR",
+  PROCESSING = "PROCESSING",
+  INTERVENTION = "INTERVENTION",
```

Comment:
Are we reusing auth's statuses because they're hardcoded in the spinner component? 
That's probably ok if we can't avoid it, although I'd probably rename this `AppVcReceiptStatus` and drop `INTERVENTION`